DESCRIPTION

```diff
@@ -112,6 +112,7 @@ Suggests:
     pls,
     PMCMR (>= 4.1),
     pROC (>= 1.8),
+    quickcheck,
```

Comment:
quickcheck is not on CRAN, I don't think that we can put it into the DESCRIPTION w/o getting into trouble on the next release.